src/test/java/kitchenpos/api/OrderApiTest.java

```diff
@@ -0,0 +1,126 @@
+package kitchenpos.api;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.sql.SQLException;
+import java.time.LocalDateTime;
+import java.util.Collections;
+import kitchenpos.dao.JdbcTemplateMenuDao;
+import kitchenpos.dao.JdbcTemplateMenuGroupDao;
+import kitchenpos.dao.JdbcTemplateOrderDao;
+import kitchenpos.dao.JdbcTemplateOrderLineItemDao;
+import kitchenpos.dao.JdbcTemplateOrderTableDao;
+import kitchenpos.domain.Menu;
+import kitchenpos.domain.MenuGroup;
+import kitchenpos.domain.Order;
+import kitchenpos.domain.OrderLineItem;
+import kitchenpos.domain.OrderTable;
+import kitchenpos.generator.MenuGenerator;
+import kitchenpos.generator.MenuGroupGenerator;
+import kitchenpos.generator.OrderGenerator;
+import kitchenpos.generator.TableGenerator;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpEntity;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+
+public class OrderApiTest extends ApiTest {
+
+    private static final String BASE_URL = "/api/orders";
+
+    @Autowired
+    private JdbcTemplateMenuGroupDao menuGroupDao;
+
+    @Autowired
+    private JdbcTemplateMenuDao menuDao;
+
+    @Autowired
+    private JdbcTemplateOrderTableDao orderTableDao;
+
+    @Autowired
+    private JdbcTemplateOrderDao orderDao;
+
+    @Autowired
+    private JdbcTemplateOrderLineItemDao orderLineItemDao;
+
+    private MenuGroup menuGroup;
+    private Menu menu;
+    private OrderTable orderTable;
+    private Order order;
+    private OrderLineItem orderLineItem;
+
+    @Override
+    @BeforeEach
+    void setUp() throws SQLException {
+        super.setUp();
+
+        menuGroup = menuGroupDao.save(MenuGroupGenerator.newInstance("두마리메뉴"));
+        menu = menuDao.save(MenuGenerator.newInstance("후라이드치킨", 16000, menuGroup.getId()));
+        orderTable = orderTableDao.save(TableGenerator.newInstance(0, false));
+        order = orderDao.save(OrderGenerator.newInstance(orderTable.getId(), "COOKING", LocalDateTime.now()));
```

Comment:
```suggestion
        order = orderDao.save(OrderGenerator.newInstance(orderTable.getId(), OrderStatus.COOKING.name(), LocalDateTime.now()));
```

기왕 있는 거, 하드코딩 하지 않는게 추후 마주하게 될 엄청난 양의 리팩토링 시에도 도움이 되지 않을까 싶습니다 :)

Reply:
좋네요! 수정하겠습니다!